### PR TITLE
Setup migrations

### PR DESCRIPTION
**Pull request by kaniak274**
#2 #6

### CODE REVIEW

File: books/models.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class Author(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+
+    first_name = db.Column(db.String(120), index=True, nullable=False)
+    last_name = db.Column(db.String(120), index=True, nullable=False)
+
+    birthday_date = db.Column(db.Date(), index=True)
+
```
Comment (KamilSupera) [inline books/models.py]: Why you separate those fields?

Comment (kaniak274) [reply books/models.py]: Why not?

Comment (kaniak274) [reply books/models.py]: I can join them if you want

Comment (KamilSupera) [reply books/models.py]: Am I ask why you did that :D

Comment (kaniak274) [reply books/models.py]: No specific reason for that

**Pull request merged**